### ISSUE TITLE
DMX effect issue with "node single color"

### ISSUE THREAD

**Issue by Phippers** [smeighan/xLights#2522]
When a DMX effect is used on a model that has the "String Type" as "Node Single Color", it does not work.

Super minor issue, I just ended up using "Single color" string type. Thought I should log it either way.

**Comment by AzGilrock** [comment]
The DMX Effect is supposed to be used with DMX Models which should all default to Single Color White node types.

**Comment by AzGilrock** [comment]
Actually I checked the code and the DMXEffect is expecting the string type to start with "Single Color" as a prefix or else it attempts to treat it as an RGB node.  I don't even know off hand what that "Node Single Color" is for or when it was added.  Maybe someone added the string type and didn't know about the naming restriction for this effect.

**Comment by AzGilrock** [comment]
I tracked down when that node type was added.  Keith just missed finding that this effect operated on the same prefix that he fixed in a couple other places.  But are you sure you really have this type of led?  Look at this issue description.  Node Single Color means you have a string where all nodes are individual addressable but can only emit a single color. https://github.com/smeighan/xLights/issues/1954

**Comment by Phippers** [comment]
I was creating a group for moving heads where the individual channels are a certain parameter for each moving head (I.E.: Name: MH Tilt, Model Type: single strand, Individual start channels for each moving head. Then, Node 1 is the Tilt value for Moving Head 2, Node 2 is Moving Head 2, etc...) So, in this configuration, it doesn't matter to me what string type it is as long as each node is a single channel. 

Thanks for the quick response, it was an easy fix for me, just wasn't sure if it was an intended feature or not.

**Comment by cybercop23** [comment]
Shuld this remain open?

**Comment by AzGilrock** [comment]
Well it was never fixed.   To be honest I’m not sure why Keith changed the DMX effect from the string contains “Single Color” to StartsWith “Single Color” unless he was specifically trying to exclude the “Node Single Color” from working in the DMXEffect.

**Comment by cybercop23** [comment]
yeah.. but 3 years old.. can' be that big of an issue. @keithsw1111 thoughts?
